docs/diagrams/AddTagSequenceDiagram.puml

```diff
@@ -84,6 +84,18 @@ activate Model
 Model --> AddTagCommand
 deactivate Model
 
-AddTagCommand --> LogicManager: commandResult(successMessage)
+create CommandResult
+AddTagCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> AddTagCommand : r
+deactivate CommandResult
+
+AddTagCommand --> LogicManager: r
 deactivate AddTagCommand
+destroy CommandResult
```

Comment:
I think CommandResult is not destroyed but returned?

Reply:
Yes thats right, will rectify this.